### PR TITLE
Torch downgrade removed from W3D3 (NeuromatchAcademy#905)

### PR DESCRIPTION
**Pull request by colleenjg**
This commit fixes issue #905. `torch 2.0` introduced an out of memory bug in the tutorial, which was addressed with commit ed8f9f2 by downgrading to `torch 1.11`. The pytorch bug was fixed in `torch 2.1` through commit [fff4a9](https://github.com/pytorch/pytorch/commit/fff4a9db8a28553ba80915fd22eec11fcd7c855e). Since Google Colab now comes with `torch 2.1` installed by default, the downgrade is no longer needed.

I also changed the supervised learning performance reported in table from `98.67%` to `98.70%`. Unlike the other methods (VAE, SimCLR, etc.), the supervised result has never been perfectly identical on each run. It varies a little bit around `98.70%` (from `98.67%` to `98.73%`, when I've run it). To avoid confusion for users, the first time this result is reported, it is now reported with a `~` (i.e., as `~98.70%`) to signal to users that their result may differ slightly from it.

### CODE REVIEW

**Comment by colleenjg** [comment]
Tagging @spirosChv to let you know about this PR reversing the `torch` downgrade you implemented a few months ago to fix a new memory bug that showed up in W3D3. :) 

**TL;DR:** `torch` on Google Colab has now been updated to `2.1`. From my tests, this fixes the memory bug you found ([as planned!](https://github.com/pytorch/pytorch/issues/104564)), and so `torch` no longer needs to be downgraded.

**Comment by spirosChv** [comment]
Fixes this https://github.com/NeuromatchAcademy/course-content-dl/issues/905
